### ISSUE TITLE
"go get -u github.com/xenolf/lego" -> exoscale provider error

### ISSUE THREAD

**Issue by ingothierack** [xenolf/lego#464]
can't current run a go get -u github.com/xenolf/lego

Looks like the exoscale-provider has an massive api change.
Known Issue?

```
#github.com/xenolf/lego/providers/dns/exoscale
../../go/src/github.com/xenolf/lego/providers/dns/exoscale/exoscale.go:58:6: unknown field 'Ttl' in struct literal of type egoscale.DNSRecord
../../go/src/github.com/xenolf/lego/providers/dns/exoscale/exoscale.go:69:9: record.Id undefined (type egoscale.DNSRecord has no field or method Id, but does have ID)
../../go/src/github.com/xenolf/lego/providers/dns/exoscale/exoscale.go:94:6: unknown field 'Id' in struct literal of type egoscale.DNSRecord
../../go/src/github.com/xenolf/lego/providers/dns/exoscale/exoscale.go:97:30: cannot use record (type egoscale.DNSRecord) as type int64 in argument to c.client.DeleteRecord
../../go/src/github.com/xenolf/lego/providers/dns/exoscale/exoscale.go:114:14: response.Record undefined (type *egoscale.DNSRecord has no field or method Record)
../../go/src/github.com/xenolf/lego/providers/dns/exoscale/exoscale.go:115:19: response.Record undefined (type *egoscale.DNSRecord has no field or method Record)

```

**Comment by xenolf** [comment]
Not known until now. Will take a look.

**Comment by xataz** [comment]
Hi,

Just for information, I'm same error on alpine, but not on archlinux

**Comment by xenolf** [comment]
Closed via #468
